### PR TITLE
Add new seata samples

### PR DESCRIPTION
**Pull request by xingfudeshi**
Hello Dubbo dev team, I am PPMC from Apache Seata. I have submitted a new Seata example that uses minimal third-party dependencies and can help beginners quickly understand how Seata and Dubbo work together.
Also, later this week, I will submit the PR of the usage documentation for seata to dubbo websites

### CODE REVIEW

**Comment by wxbty** [comment]
Thanks for your pr ，@xingfudeshi， I don't quite understand why the demo needs to be completely replaced.  The existing one looks simple enough, can you list some?

**Comment by xingfudeshi** [comment]
Hi, @wxbty 
The seata community received an email from the dubbo community, hoping that we can help update the samples and documents for the upcoming Apache Dubbo 3.3.0. So, I checked the documents and examples of seata that already exist in the dubbo community, and I found some problems. :
1. The document is a bit complicated and contains some repeated information, which is more theoretical.
2. For Dubbo users, they may not be familiar with Seata, so it is necessary that we simplify cases and tutorials to help them quickly experience how Seata and Dubbo work.

Therefore, I re-wrote a document, and the PR has been submitted here [2], and this sample is matched with this document. This time the document does not rely on any third-party components, and only requires dubbo and seata to complete, I believe It will definitely give users a good experience.

[1]https://lists.apache.org/thread/6853sjp0qxt2rv975fbkhvf4rpo1fsnv
[2]https://github.com/apache/dubbo-website/pull/2906
doc preview:https://deploy-preview-2905--dubbo.netlify.app/zh-cn/overview/mannual/java-sdk/tasks/trasaction/distributed-transaction/
​

**Comment by AlbumenJ** [comment]
@slievrly @chickenlj PTAL

**Comment by xingfudeshi** [comment]
> Okay, I've got the background, I will check it later. In addition, if you use dubbo-sample examples, it is recommended that the document add one-click test examples: ./test/run-tests.sh 2-advanced/dubbo-samples-seata. It will start the relevant components and services, start it once, and finally execute the test case

Ok.I'll do that.

**Comment by wxbty** [comment]
@xingfudeshi   Please refer to the original example to restore the two files case-configuration.yml and case-versions.conf. The example of dubbo-sample not only allows users to manually execute unit tests, but also is the source of automatic integration testing for dubbo main repository workflow. Automatic testing requires One-click testing, this is the role of case-configuration.yml. And case-versions.conf can help us test the dependencies of the specified version matrix

**Comment by xingfudeshi** [comment]
> @xingfudeshi Please refer to the original example to restore the two files case-configuration.yml and case-versions.conf. The example of dubbo-sample not only allows users to manually execute unit tests, but also is the source of automatic integration testing for dubbo main repository workflow. Automatic testing requires One-click testing, this is the role of case-configuration.yml. And case-versions.conf can help us test the dependencies of the specified version matrix

Done.

File: 2-advanced/dubbo-samples-seata/case-configuration.yml [hunk 1]
```diff
@@ -80,76 +67,61 @@ services:
...
       - seata-mysql
       - seata-server
-      - account-service
 
   business-test:
     type: test
     basedir: dubbo-samples-seata-business
     tests:
-      - "**/*IT.class"
+      - "**/*Tests.class"
```
Comment (wxbty) [inline 2-advanced/dubbo-samples-seata/case-configuration.yml:111]: In dubbo-sample, *IT.class is a unified suffix, which can facilitate narrative, pls restore this.

**Review by wxbty — CHANGES_REQUESTED** [review]

File: 2-advanced/dubbo-samples-seata/case-configuration.yml (hunk 1 above)
Comment (xingfudeshi) [reply 2-advanced/dubbo-samples-seata/case-configuration.yml:111]: OK

**Comment by wxbty** [comment]
[root@VM-16-12-centos dubbo-samples]# ./test/run-tests.sh 2-advanced/dubbo-samples-seata/

File: 2-advanced/dubbo-samples-seata/case-configuration.yml (hunk 1 above)
Comment (xingfudeshi) [reply 2-advanced/dubbo-samples-seata/case-configuration.yml]: Done

**Comment by xingfudeshi** [comment]
I tried running these samples locally, but I still got stuck here.
image
image

**Comment by xingfudeshi** [comment]
I have java17 installed,but still get this
image
image
image

**Comment by xingfudeshi** [comment]
@wxbty

**Comment by wxbty** [comment]
What about changing to java8 version?

**Comment by xingfudeshi** [comment]
> What about changing to java8 version? Java17 seems to have a problem with the old seata-sample, and the test is ignored during execution.

Ok.I'll take a look on it.

**Comment by xingfudeshi** [comment]
I have been trying for a long time but still cannot succeed. Please help me check where the problem is
It seems to be a network issue in the Docker environment. I'm not sure if it's because I'm using Multicast. Please take a look at my code.
```
dubbo:
    application:
        logger: slf4j
        name: ${spring.application.name}
        qos-enable: false
    registry:
        address: multicast://224.5.6.7:1234
    protocol:
        port: 20881
        name: dubbo
```
I don't want to introduce any extra middleware to increase the time cost of user experience data, so I use Multicast as the registry.

**Comment by xingfudeshi** [comment]
image
image
image

image
image

**Comment by xingfudeshi** [comment]
I can run these samples through a regular spring boot application without any problem

**Comment by xingfudeshi** [comment]
I've figured it out, the previous issue was due to some mistakes.Now the test has passed.
image

**Comment by wxbty** [comment]
On my server, it also passed the test and there seems to be no problem.  @AlbumenJ   PTAL again

**Review by wxbty — APPROVED** [review]
LGTM

**Review by AlbumenJ — APPROVED** [review]

**Review by AlbumenJ — COMMENTED** [review]
@xingfudeshi PTAL

**Pull request merged**